tests/coverage/README.md

```diff
@@ -0,0 +1,218 @@
+# Guiding contributors, reviewers & maintainers through the complexity of SpeechBrain testing.
```

Comment:
Ok we MUST have a discussion to re-localize properly all this stuff. It must be easier to reach than just in tests/ where users will naturally avoid to go. This should be put in the README somewhere (learn how to blabla) or in the documentation.